airbyte-integrations/connectors/source-auth0/integration_tests/configured_catalog.json

```diff
@@ -38,8 +36,7 @@
         "supported_sync_modes": ["full_refresh"]
       },
       "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite",
-      "primary_key": [["id"]]
```

Comment:
why are we removing these values?

Reply:
Because the streams doesn't output id, primary key is different